src/main/kotlin/calculator/operation/Operator.kt

```diff
@@ -7,23 +7,17 @@ enum class Operator(val symbol: String) {
     DIV("/");
 
     companion object {
-        val OPERATOR_REGEX = "[+\\-*/]".toRegex()
-        val OPERAND_REGEX = "\\d".toRegex()
-        val OPERATOR_AND_OPERAND_REGEX = "[\\d+\\-*/]+".toRegex()
+        private val OPERATOR_SYMBOLS = Operator.values().map { it.symbol }.toSet()
 
-        fun splitOperators(expression: String): List<String> =
-            expression
-                .replace(OPERAND_REGEX, "")
-                .asSequence()
-                .map { it.toString() }
-                .toList()
+        fun isOperator(value: String): Boolean = OPERATOR_SYMBOLS.contains(value)
 
-        fun splitOperands(expression: String): List<String> =
-            expression
-                .split(OPERATOR_REGEX)
-                .asSequence()
-                .filter { it.isNotBlank() }
-                .toList()
+        fun splitOperators(expression: String): List<String> {
+            return expression.filter { it.toString() in OPERATOR_SYMBOLS }.map { it.toString() }
```

Comment:
```suggestion
            return expression
                .asSequence()
                .filter { it.toString() in OPERATOR_SYMBOLS }
                .map { it.toString() }
```
지금과 같이 처리 단계가 두 단계 이상(filter + map)인 경우에는 sequence를 활용해주시면 좋습니다.
filter를 사용해도 새로운 컬렉션을 만들어내고, map을 사용해도 새로운 컬렉션을 만들어냅니다.
해서, 총 두 개의 새로운 컬렉션을 만들게 됩니다.
여기서 sequence를 사용하면 하나의 컬렉션만을 만들게 됩니다. 😄  (이하 관련 내용 동일)